Firmware/ultralcd.cpp

```diff
@@ -1929,8 +1919,16 @@ void mFilamentItem(uint16_t nTemp, uint16_t nTempBed)
             // handled earlier
             break;
         }
-        if (bFilamentWaitingFlag) Sound_MakeSound(e_SOUND_TYPE_StandardPrompt);
-        bFilamentWaitingFlag = false;
+        if (bFilamentWaitingFlag) {
```

Comment:
:thinking: how was the previous code indented? There has been no `{`, it looks like only `Sound_MakeSound` was executed `if (bFilamentWaitingFlag)`